lib/node/index.js

```diff
@@ -42,5 +42,7 @@ export {
   // Make FileUrlStorage module available as it will not be set by default.
   FileUrlStorage,
   canStoreURLs,
-  enableDebugLog
+  enableDebugLog,
+  HttpStack,
```

Comment:
:+1: Can you also add an export for the browser's HttpStack?

Reply:
Don't merge just yet. I'll try add a browser's HttpStack also, but it may take a little more time.